.github/workflows/label.yaml

```
@@ -0,0 +1,19 @@
+name: Pull Request Labeler
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request_target:
+    branches:
+      - "main"
+
+jobs:
+  labeler:
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/labeler@v5
```

Auto-include paths starting with dot:

```suggestion
    - uses: actions/labeler@v5
      with:
        dot: true
```